DataStorageEngine/DSECommonSenseProxy.h

```diff
@@ -0,0 +1,172 @@
+//
+//  DSECommonSenseProxy.h
+//  SensePlatform
+//
+//  Created by Joris Janssen on 16/08/15.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+/**
+ *	The DSECommonSenseProxy class consists of static methods that wrap the API so that it is more convenient to call. 
+ 
+	It is up to the user to either call the API in a synchronous or asynchronous manner. Every API call will have two callback methods that get called whenever the call returns. The success method will be called when the API call was successfull. The failure call will be called when the API call was not successfull.
+ 
+		- In cases where the API call was successfull, resulting data will be passed on to the success callback functions as parameters. Please see indivdual functions for details.
+ 
+		- The failure callback will be called with an NSError object as a parameter. The object can be used to get details about the error by parsing the response object. Specifically, the NSError response code will correspond to the code that was received from the server. The NSError response message will correspond to the message that was received from the server.
+ 
+	
+	The DSECommonSenseProxy class does not now about DataStorageEngine concepts, it simply wraps the API that is available for CommonSense. Any translations between concepts in the DataStorageEngine and CommonSense should be made by other objects (most notably the difference between Source and Device). 
+ 
+	The DSECommonSenseProxy uses a Session ID and Application Key to get authorized for API access. These parameters are managed automatically but can be set by others as well. -->> do we want this?
```

Comment:
Application key is used by the Coaching Engine. As far as I know, for CommonSense we only need a session-id